web-partner-api/example.html

```diff
@@ -6,23 +6,95 @@
     <meta content="width=device-width,initial-scale=1" name="viewport">
     <meta charset="UTF-8">
     <title>MoveApps Web-Partner API example</title>
+    <link rel=“stylesheet” href=“https://cdn.jsdelivr.net/gh/jgthms/minireset.css@master/minireset.min.css“>
+    <style>
+        body {
+            font-family: sans-serif;
+        }
+    </style>
 </head>
 
 <body>
     <h1>API for analysis output files functional</h1>
     <p>Try out integrating files that you have produced with a MoveApps workflow into your own personal webpage! They are then automatically updated when scheduling workflow runs.</p>
 
+    <section>
+        <p>The resources provided by the API are protected. You can submit the credentials by a) HTTP basic access authentication or b) an API key.</p>
+        <p>
+            Keep in mind that the tokens and/or API-Keys will be <b>visible to the public</b>.
+            That is okay for workflows which generate public resources (aka artifacts).
+        </p>
+        <p>
+            If your workflow produces <b>sensitive resources</b> consider consuming the artifacts by a own backend system in order to hide this token from the public.
+        </p>
+        <table border="1">
+            <tr>
+                <th />
+                <th>Basic Auth</th>
+                <th>API key (URL encoded!)</th>
+            </th>
+            <tr>
+                <th>User</th>
+                <td colspan="2" style="text-align: center;"><code id="username"></code></td>
+            </tr>
+            <tr>
+                <th>Token</th>
+                <td><code id="apiToken"></code></td>
+                <td><code id="apiTokenUrlEncoded"></code></td>
+            </tr>
+            <caption>The credentials for this example page. You need to register your own ones at MoveApps.</caption>
+        </table>
+
+        <h2>Preparation</h2>
+        <ol>
+            <li>Find the <em>OUTPUT</em> Button at your Workflow-Instance Screen on <a href="https://www.moveapps.org">MoveApps</a></li>
+            <li>In the <em>workflow Instance Output</em></li> Screen localize and press the button <em>API ACCESS</em>
+            <li>Press <em>REGISTER</em></li>
+            <li>Note values of <code>Username</code> and <code>Password</code> (aka Token)</li>
+            <li>Localize and press the link on this screen names <em>this overview</em></li>
+            <li>On this overview all the files produced by your workflow-instance are listed. Use these links to some files for the next sections.</li>
+        </ol>
+        <p> The <a href="http://localhost:8080/web-partner/v1/workflowInstances/d8fcb45e-412d-4002-81d6-0fcbdbd5fa5d/results/index?api-key=cZXqvA45X8LHVME7515CbIybg4dQX4wl">mentioned overview for this example</a> (use something that can parse XML!).</p>
```

Comment:
TODO Clemens: as soon as the sprint reaches `www` replace all `localhost` links w/ `www`